api/controllers/auth.js

```diff
@@ -44,6 +50,48 @@ const registration = async (req, res, next) => {
   }
 };
 
+const reset = async (req, res, next) => {
+  const { email } = req.body;
+  try {
+    const v = await emailSchema.validate({ email });
+    const a = v && (await services.resetPasswordRequest({ email }));
+    let status = a.status ? a.status : 200;
+    let response = a.messageId ? 'Mail was sent' : a.message;
+    res.status(status).send(response);
+  } catch (e) {
+    next(e);
+  }
+  //
+};
+
+const resetApprovementPassword = async (req, res, next) => {
```

Comment:
get rid of next in each function. next func usually is using for making usual endpoints as middleware